src/drivers/mct/cime_config/config_component.xml

```diff
@@ -136,7 +136,7 @@
     <desc>Model grid - DO NOT EDIT (for experts only)</desc>
   </entry>
 
-  <entry id="SUPPORTED_BY">
+  <entry id="MACHINE_SUPPORTED_BY">
```

Comment:
There is only one SUPPORTED_BY field.  Choose one of these and delete the other.